packages/vps/packaging

```diff
@@ -0,0 +1,17 @@
+set -e -x
+
+if [ -z "$BOSH_PACKAGES_DIR" ]; then
+	pkg_dir=$(readlink -nf /var/vcap/packages/golang_1.7)
+else
+	pkg_dir=$BOSH_PACKAGES_DIR/golang_1.7
+fi
+
+export GOROOT=$pkg_dir
+export GOPATH=$PWD
+export PATH=$GOROOT/bin:$PATH
+
+mkdir ../src && cp -a * ../src/ && mv ../src ./src
```

Comment:
how do you know you are in correct directory? Seems to me `mkdir ../src ...` is a bit dangerous? Is there a way to guarantee we are in correct start directory?

Look at the [Zookeeper release](https://github.com/cppforlife/zookeeper-release) [`packaging`](zookeeper-release/packages/java/packaging) scripts as examples... but other releases should have that too.